src/Pact/Analyze/Remote/Server.hs

```diff
@@ -122,7 +122,7 @@ moduleNotFoundP = MP.string "<interactive>:"
                *> digitsP *> MP.char ':'
                *> digitsP *> MP.char ':'
                *> MP.string " Module \""
-               *> fmap (ModuleName . T.pack) (MP.some $ MP.notChar '"')
+               *> fmap (\n -> ModuleName (T.pack n) Nothing) (MP.some $ MP.notChar '"')
```

Comment:
This is remote stuff, so i'm not sure how we can get namespaces here. Must discuss